pkgs/applications/virtualization/railcar/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub, fetchpatch, rustPlatform, libseccomp }:
+
+rustPlatform.buildRustPackage rec {
+  name = "railcar=${version}";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "oracle";
+    repo = "railcar";
+    rev = "v${version}";
+    sha256 = "09zn160qxd7760ii6rs5nhr00qmaz49x1plclscznxh9hinyjyh9";
+  };
+  
+  cargoSha256 = "16f3ys0zzha8l5jdklmrqivl8hmrb9qgqgzcm3jn06v45hls9lan";
+
+  buildInputs = [ libseccomp ];
+  cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
Would it be a good idea to submit the patch upstream?

Reply:
See https://github.com/oracle/railcar/pull/44